handbook/people-ops/spending-company-money.md

```diff
@@ -1,12 +1,14 @@
 # Spending company money
 
-We want to make sure you have what you need to be productive and happy in your role at Sourcegraph, so we've come up with a few guidelines to help you when you have work-related expenses. Below you'll find a list of some typical purchases that would be reimbursed for, as well as a few things to keep in mind when expensing purchases:
+We want to make sure you have what you need to be productive and happy in your role at Sourcegraph, so we've come up with a few guidelines to help you when you have work-related expenses. Below you'll find a list of some typical purchases that you would be reimbursed for, as well as a few things to keep in mind when expensing purchases:
 
 - You are free to purchase these items and expense them without asking permission as long as it's in the interest of the company. When in doubt, please contact @sourcegraph/people-ops for clarification before the purchase.
 - You likely won't need every single item on this list—please use your best judgment and buy when you need them. Remember, Sourcegraph doesn't have unlimited funds—spend this money as if it was your own.
   - For example, if you intend to work out of a coworking space most of the time, you may not need a fully equipped home office as well.
-- We have an "all-in" total budget of $1000 for desk setup, which includes all the items in the "Desk set-up expenses" section (with the exception of laptops). If you expect to significantly exceed the listed amount on any purchase, please notify @sourcegraph/people-ops so we can decide if we need to update this guide.
-- Laptops and laptop insurance are fully reimbursable outside of the $1000 "all-in" budget, however, engineering roles have different hardware requirements from other roles (please see below for necessary specs corresponding to your role).
+- We have two separate "all-in" budgets (in USD), one for full-time teammates and one for interns:
+  - The budget for interns is $500 and includes only the electronic items in the "Desk set-up expenses" section (with the exception of laptops, as these are separate from the all-in budget).
+  - The budget for full-time teammates is $1000 for your desk setup, which includes all the items in the "Desk set-up expenses" section (with the exception of laptops, as these are separate from the all-in budget). If you expect to significantly exceed the listed amount on any purchase, please notify @sourcegraph/people-ops so we can decide if we need to update this guide.
+- Laptops and laptop insurance are fully reimbursable outside of the $1000 all-in budget, however, engineering roles have different hardware requirements from other roles (please see below for necessary specs corresponding to your role).
```

Comment:
Perhaps lead with this bullet (before the "We have two separate..." bullet), so you don't have to caveat the two sub-bullets on totals quite so much? I just feel like this is a confusing section, and might take a few readings to clarify...

Or perhaps we pull the laptops out of the "Desk set-up expenses" header entirely, and put them in their own section? That could reduce some of the confusion.

Reply:
Made a few changes to make it more clear!